### ISSUE TITLE
Create DOCKERFILE for implementing a local dev environment

### ISSUE THREAD

**Issue by t0xic0der** [t0xic0der/fedora-easyfix#24]
Create DOCKERFILE for implementing a local dev environment

**Comment by khallnayak** [comment]
Can you please specify the details and it would help to remove the ambiguity by explaining with an example.

**Comment by t0xic0der** [comment]
Here's an example https://pagure.io/fedora-websites/blob/master/f/Dockerfile. The container would help open a siloed hosting environment by fetching requirements and starting the server without tainting the user's development environment.

**Comment by t0xic0der** [comment]
Here's another example that can act as a reference https://github.com/t0xic0der/fragment/pull/33.

**Comment by khallnayak** [comment]
Thanks for the clarification.

**Comment by khallnayak** [comment]
I cloned the repo into a python:3.8-alpine container and installed requirements and this happened
```
/fedora-easyfix # index-easyfix-issues 
Traceback (most recent call last):
  File "/usr/local/bin/index-easyfix-issues", line 33, in <module>
    sys.exit(load_entry_point('fedora-easyfix==0.3.0', 'console_scripts', 'index-easyfix-issues')())
  File "/usr/local/bin/index-easyfix-issues", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/usr/local/lib/python3.8/importlib/metadata.py", line 77, in load
    module = import_module(match.group('module'))
  File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.8/site-packages/fedora_easyfix-0.3.0-py3.8.egg/fedora_easyfix/utilities/producer.py", line 35, in <module>
    from yaml import CLoader, load
ImportError: cannot import name 'CLoader' from 'yaml' (/usr/local/lib/python3.8/site-packages/PyYAML-5.4.1-py3.8-linux-x86_64.egg/yaml/__init__.py)
```

**Comment by t0xic0der** [comment]
Could you check if the same issue persists with other base images? This seems to be an issue at the dependencies' end.

**Comment by khallnayak** [comment]
The issue resolves for python version > 3.8 (I just checked it on 3.9.7)
